onadata/apps/logger/data_migration/xformtree.py

```diff
@@ -175,6 +211,12 @@ def added_fields(self, other_tree):
             if field not in other_tree.get_fields()
         ]
 
+    def remained_fields(self, other_tree):
+        """Return fields that remained in tree compared to other tree"""
+        return list(
+            set(self.get_fields()) - set(self.added_fields(other_tree))
+        )
+
```

Comment:
This looks unused, plus a bit confusing as to what fields it contains

Reply:
Removed